RecDP/tests/test_llmutils_pipelines.py

```diff
@@ -184,3 +184,25 @@ def test_pipeline_execute_pandasdf_spark(self):
         ret = pipeline.execute(df)
         display(ret.toPandas())
 
+    def test_llm_rag_pipeline(self):
+        from pyrecdp.primitives.operations.text_embedding import LangchainDocumentEmbedding as DocumentEmbedding
+        model_root_path = os.path.join(RECDP_MODELS_CACHE, "huggingface")
+        pipeline = TextPipeline()
+        ops = [
+            DirectoryLoader("tests/data/llm_data/document", glob="**/*.pdf"),
+            DocumentSplit(),
```

Comment:
Why use DocumentSplit not LangchainTextSplit. What is the major difference of these two Splitter